lib/elixir/test/elixir/string_test.exs

```diff
@@ -580,11 +581,11 @@ defmodule StringTest do
     assert String.chunk("ødskfjあ\x11ska", :valid)
            == ["ødskfjあ\x11ska"]
     assert String.chunk("abc\u{0ffff}def", :valid)
-           == ["abc", <<0x0FFFF::utf8>>, "def"]
+           == ["abc" <> <<0x0FFFF::utf8>> <> "def"]
     assert String.chunk("\u{0FFFE}\u{3FFFF}привет\u{0FFFF}мир", :valid)
```

Comment:
We can remove this example and the next one, as they are effectively not adding new information anymore.